### ISSUE TITLE
NVDA does not read checkbox unchecked/checked if it is wrapped inside div with role as gridcell.

### ISSUE THREAD

**Issue by vivek31be** [nvaccess/nvda#9051]
URL - https://www.w3schools.com/code/tryit.asp?filename=FY4O6VWSYQLE

if using tab key focus on first checkbox and then check/uncheck using space key. NVDA will read it as space only.

NVDA should read as checked/unchecked as for other checkbox in the test url.

**Comment by DrSooom** [comment]
@vivek31be: Please completely fill out the bug draft. Your issue description contains too less information at the moment. Thanks.

**Comment by leonardder** [comment]
I'm afraid I Have to close this as invalid due to the following reasons:

1. The gridcell in the example isn't part of any grid. Furthermore, it has a tabindex of 0. The first item in the tab order is the grid cell itself, anoter tab brings me to the check box.
2. There are labels for the checkboxes, but they aren't properly assigned to the checkboxes. This is why NVDA only reads the checkbox role and its checked/unchecked state when using tab to navigate the page with browse mode off.
3. Relevant details, including the browser(s) you tested with, are missing.

**Comment by leonardder** [comment]
Actually a works for me, since tab brings me to the check box, though not in first instance.
